docker/centos6/Dockerfile

```diff
@@ -239,6 +240,45 @@ RUN source /opt/rh/devtoolset-8/enable && \
     mkdir -p /opt/boringssl/lib && \
     cp crypto/libcrypto.a ssl/libssl.a /opt/boringssl/lib/
 
+# build/install wolfssl
+RUN source /opt/rh/devtoolset-8/enable
```

Comment:
The RUN directive will only take a single string. If you look at other install stanzas, you can see that there is a `&& \` at the end of each line. This will need to follow this pattern.